AAP - some fixes related to CSV export (SB-451)

pmauduit opened the pull request:
Following my previous work on AAP, some fields were missing the CSV export (see https://jira.swisstopo.ch/browse/GEOCAT_SB-451 for the details).

Tests: adapted related testsuite, tested the current branch onto tc-geocat.dev

Vampouille commented:
For testing, a MD must have "AAP-Bund" as keyword see : https://github.com/geoadmin/geocat/commit/c9051778c84b4ee5113002ca4435423234038d1f

pmauduit commented:
Rebased and added a field in the export (topicCategory). Waiting for customer's feedback (still don't know if we have to take the first gmd:topicCategoryCode or concatenate every fields).
